js/loginform.js

```diff
@@ -0,0 +1,84 @@
+const emailInput = document.querySelector("#email");
+const emailInputBox = document.querySelector(".input-email");
+const passwordInput = document.querySelector("#password");
+const passwordInputBox = document.querySelector(".input-password");
+const loginButton = document.querySelector(".login-button");
+const nicknameInput = document.querySelector("#nickname");
+const nicknameInputBox = document.querySelector(".input-nickname")
+const passwordConfirmInput = document.querySelector("#password-confirm");
+const passwordConfirmInputBox = document.querySelector(".input-password-confirm");
+const signupLoginButton = document.querySelector(".signup-login-button");
+const togglePasswordButton = document.querySelector("#toggle-password-visibility");
+const eyeIcon = document.querySelector("#eye-icon");
+
+// Email 유효성 검사
+function checkEmail(email) {
+    const emailPattern = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-za-z0-9\-]+/;
+    return emailPattern.test(email);
+};
+
+// Email 미입력 시 && Email 형식 오류 시
+emailInput.addEventListener("focusout", function(e){
+    if(!e.target.value){
+        emailInputBox.classList.add("error");
+        emailInputBox.classList.remove("errorform");
+    }else{
+        emailInputBox.classList.remove("error");
+        if (!checkEmail(e.target.value)){
+            emailInputBox.classList.add("errorform");
+        } else {
+            emailInputBox.classList.remove("errorform");
```

Comment:
```suggestion
            emailInputBox.classList.toggle("errorform");
```
위의 메서드 확용해보면 좋을 거 같아요~